02-express-tutorial/13-router-people.js

```diff
@@ -0,0 +1,22 @@
+const express = require('express')
+const router = express.Router()
+
+const {
+  getPeople,
+  createPerson,
+  createPersonPostman,
+  updatePerson,
+  deletePerson,
+} = require('../controllers/people')
+
+// router.get('/', getPeople)
+// router.post('/', createPerson)
+// router.post('/postman', createPersonPostman)
+// router.put('/:id', updatePerson)
+// router.delete('/:id', deletePerson)
```

Comment:
Good work.  A common question in job interview - design simple CRUD  endpoints for REST API.